### DISCUSSION TITLE
Kyocera 4054ci Wia

### DISCUSSION THREAD

**Discussion by kratzerph** [cyanfish/naps2#95] [Support]
**Describe the bug**
Kyocera 4054ci error if i press scan
**To Reproduce**
1. Add Scanner to profile
2. click on Scan
3. See error

```
APS2.Scan.Exceptions.ScanDriverUnknownException: Ein Fehler ist mit dem Scanner-Treiber aufgetreten. ---> NAPS2.Scan.Wia.Native.WiaException: WIA error code 80004005
   bei NAPS2.Scan.Wia.Native.WiaTransfer.Download()
   bei NAPS2.Scan.Wia.WiaScanOperation.DoTransfer(Concrete source, WiaDevice device, WiaItem item)
   bei NAPS2.Scan.Wia.WiaScanOperation.Scan(Concrete source)
   bei NAPS2.Scan.Wia.WiaScanOperation.<>c__DisplayClass26_0.<Start>b__0()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei NAPS2.Scan.Wia.WiaScanErrors.ThrowDeviceError(WiaException e)
   bei NAPS2.Scan.Wia.WiaScanOperation.<>c__DisplayClass26_0.<Start>b__0()
   bei Microsoft.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   bei Microsoft.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccess(Task task)
   bei NAPS2.Scan.ScanDriverBase.<>c__DisplayClass30_0.<<Scan>b__0>d.MoveNext()
   bei Microsoft.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   bei Microsoft.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccess(Task task)
   bei NAPS2.Scan.ScanPerformer.<PerformScan>d__7.MoveNext()

```

**Expected behavior**
Scan should begin,
With Windows Building Fax/Scan app scanner works fine

**Comment by cyanfish** [comment]
Hi, I recommend trying the steps on the [Troubleshooting](https://www.naps2.com/doc/troubleshooting#scan-error) page.

**Reply by hansaliyad1** [reply]
@cyanfish I tried troubleshooting steps. Did restart the scanner and the computer. The drivers were freshly installed. I also tried TWAIN driver but that gives `TWAIN error: LowMemory` error. Will greatly appreciate any feedback. Thank you!

**Comment by gcardoso22** [comment]
Did you managed to find a solution for this? Having the same issue on a Kyocera M2040dn

**Comment by neshkom** [comment]
Hi. when I try to scan document with wia driver (scanner is TASKALFA 5053cl), I receive following message

NAPS2.Wia.WiaException: WIA error code 80004005
   at bool NAPS2.Wia.WiaTransfer.Download()
   at void NAPS2.Scan.Internal.Wia.WiaScanDriver+WiaScanContext.DoTransfer(WiaDevice device, WiaItem item)
   at async Task NAPS2.Scan.Internal.Wia.WiaScanDriver+WiaScanContext.Scan(WiaVersion wiaVersion)
   at async Task NAPS2.Scan.Internal.Wia.WiaScanDriver.Scan(ScanOptions options, CancellationToken cancelToken, IScanEvents scanEvents, Action<IMemoryImage> callback)+(?) => { }

Is there any idea how to solve this problem?

Thank in advance
